autopush/webpush_server.py

```diff
@@ -408,15 +413,16 @@ def lookup_user(self, hello):
         flags["check_storage"] = True
 
         # Determine if message table rotation is needed
-        flags["message_month"] = record["current_month"]
```

Comment:
we still need to continue migrating to the new pegged table, so revert this. my bad!